### PR TITLE
[zh] Add contributing/_index.md

### PR DESCRIPTION
**Pull request by freezer712**
Add content/zh/docs/contributing/_index.md

### CODE REVIEW

File: content/zh/docs/contributing/_index.md [hunk 1]
```diff
@@ -0,0 +1,572 @@
+---
+title: 贡献
+description: 了解如何为OpenTelemetry文档做贡献.
+aliases: [/docs/contribution-guidelines]
```
Comment (chalin) [inline content/zh/docs/contributing/_index.md:4]:
```suggestion
```

**Review by chalin — COMMENTED** [review]
Drive-by comment

**Comment by svrnm** [comment]
@freezer712 thanks for your contribution! Can you take a look at the CLA and sign it? If you have any question around it or any concerns do not hesitate to let us know

File: content/zh/docs/contributing/_index.md (hunk 1 above)
Comment (freezer712) [reply content/zh/docs/contributing/_index.md]: yes,this should be removed. I have removed it in the latest commit with other changes together, thank you~

**Comment by windsonsea** [comment]
Plase sign the CLA.

**Comment by freezer712** [comment]
Sorry for the delayed reply. I encountered some problems when processing cla. At the same time, in order to make the commit more concise and clear, I re-submitted a PR. For details, please see #4667

**Comment by freezer712** [comment]
> @freezer712 thanks for your contribution! Can you take a look at the CLA and sign it? If you have any question around it or any concerns do not hesitate to let us know

thanks, I have resolved the problem in a new PR.

**Pull request closed without merging**